service/cloudFormation/sam-template-empty.yml

```diff
@@ -18,7 +18,7 @@ Resources:
       Description: Lambda Function to run playwright test
 #add name of the text
       FunctionName: string
-      MemorySize: 512
+      MemorySize: 1024
       Timeout: 60
```

Comment:
In the template, the timeout is 60, and in the code for creating the lambda function, the timeout is 300. Is this intentional?